src/main/scala/zio/json/decoder.scala

```diff
@@ -235,6 +235,19 @@ object Decoder extends GeneratedTuples with DecoderLowPriority1 with DecoderLowP
       builder(trace, in, new mutable.ListBuffer[A])
   }
 
+  implicit def vector[A: Decoder]: Decoder[Vector[A]] = new Decoder[Vector[A]] {
+    def unsafeDecode(trace: List[JsonError], in: RetractReader): Vector[A] =
+      builder(trace, in, new mutable.ListBuffer[A]).toVector
+  }
+
+  // FIXME compiles without seq but tests fails
+  // assertion failed: Right(Monster(List())) != Right(Monster(List(5XL, 2XL, XL)))
+  // maybe sth is off with cbf ?
+  implicit def seq[A: Decoder]: Decoder[Seq[A]] = new Decoder[Seq[A]] {
```

Comment:
I don't think we'd want `seq` in the decoder, if something is wrong with CBF then it should be fixed.